web_accounts_list.json

```diff
@@ -105,6 +105,17 @@
          "category" : "business",
          "valid" : true
       },
+      {
+         "name" : "Behance",
+         "check_uri" : "https://www.behance.net/{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : "<title>Behance</title>",

```

Comment:
When I look at Alice's profile in the site, I get `<title>Alice Wu on Behance</title>` instead of the title string noted above. Can you check it please and adjust?

Reply:
You will see that in the source code of the page after it is loaded, and the browser has had the chance to run all the scripts, performed redirects et cetera. If you simply curl the initial page, you will see the difference. Tested it, it is 100% correct.

Reply:
I see what you mean....does the python checker also see the site like the curl request does?